nano/node/json_handler.cpp

```diff
@@ -2069,19 +2069,35 @@ void nano::json_handler::database_txn_tracker ()
 void nano::json_handler::delegators ()
 {
 	auto account (account_impl ());
+	auto count (count_optional_impl ());
```

Comment:
Probably can have default value like 1024 or something (i.e. `auto count (count_optional_impl ());`). By default it's 2^64-1